playlist_souffle/entities/track.py

```diff
@@ -0,0 +1,17 @@
+"""Module for the track entity."""
+
+
+def extract_track_collection_id(track, collection_type):
+    """Extract the collection id of a track for the given collection_type.
+
+    >>> from playlist_souffle.definitions.track import Track
+    >>> my_track = Track(id='TRACK_ID', artist='ARTIST_ID', album='ALBUM_ID')
+
+    >>> extract_track_collection_id(my_track, 'artist')
+    'ARTIST_ID'
```

Comment:
aha. just realized that it's not actually returning the string `ARTIST_ID` and it's just an example.

Reply:
Some comments like this are really confusing. What exactly are you saying?